src/components/NcModal/NcModal.vue

```diff
@@ -455,6 +455,14 @@ export default {
 			default: false,
 		},
 
+		/**
+		 * Make the modal header appear dark.
+		 */
+		darkHeader: {
```

Comment:
This should not only change the header but the overall backdrop.
Meaning in this case also the backdrop should be bright.

So probably nicer to have: `brightBackdrop` prop with default `false`

Reply:
Implemented now with @susnux